src/apps/Settings/Settings.js

```diff
@@ -87,16 +149,90 @@ class Settings extends React.Component {
     const { cache, currencies } = this.props
     this.propagateSelectedCurrency(cache, currencies[index])
   }
+  handleDepositTestTokens = () => {
+    const { selectedTestToken, testTokens } = this.state
+    const { account } = this.props
+    const tokenAddress = testTokens[selectedTestToken]
+
+    const { apps } = this.props
+    const finance = apps.find(app => app.appId === appIds.Finance)
+    if (!finance || !finance.proxyAddress) {
+      return
+    }
+
+    airdrop(getWeb3(web3Providers.wallet), finance.proxyAddress, account)
+  }
+  handleTestTokenChange = index => {
+    this.setState({
+      selectedTestToken: index,
+    })
+  }
   propagateSelectedCurrency(cache, selectedCurrency) {
     cache.update(CACHE_KEY, settings => ({ ...settings, selectedCurrency }))
   }
   render() {
-    const { currencies, daoAddr, network, selectedCurrency, apps } = this.props
+    const {
+      currencies,
+      daoAddr,
+      account,
+      network: userNetwork,
+      selectedCurrency,
+      apps,
+    } = this.props
+    const { selectedTestToken, testTokens } = this.state
+    const testTokenSymbols = testTokens.map(
+      ({ symbol, name }) => `${name} (${symbol})`
+    )
+
+    const enableTransactions = !!account && userNetwork === network.type
     return (
       <Main>
         <StyledAppBar title="Settings" />
         <ScrollWrapper>
           <Content>
+            <Option
+              name="Testing Tokens"
+              text="Deposit some tokens into your organization for testing purposes."
+            >
+              <div>
+                <Text color={theme.textSecondary} smallcaps>
+                  Select Token
+                </Text>
+                <FieldTwoParts>
+                  <DropDown
+                    active={selectedTestToken}
+                    items={testTokenSymbols}
+                    onChange={this.handleTestTokenChange}
+                    style={{ width: '400px' }}
+                    wide
+                  />
+                  {enableTransactions ? (
+                    <Button
+                      mode="secondary"
+                      onClick={this.handleDepositTestTokens}
+                      style={{ marginLeft: '10px' }}
+                    >
+                      Get tokens
+                    </Button>
+                  ) : (
+                    <Text size="small" style={{ marginLeft: '10px' }}>
+                      {(() => {
+                        if (userNetwork !== network.type) {
+                          return `Please select the ${network.type} network in MetaMask.`
+                        }
+                        return `Please unlock your account in MetaMask.`
```

Comment:
Maybe we should have a component for this since it's both on onboarding and in here (and the signer too?)? Not critical, but would be nice to do after launch.